### ISSUE TITLE
symling to afl-clang did not work in configure step

### ISSUE THREAD

**Issue by toralf** [AFLplusplus/AFLplusplus#389]
At a hardened Gentoo
```export CC="/usr/bin/afl-clang"```
didn't worked with Tor sources ("configure: error: C compiler cannot create executables"), whereas
```export CC="/usr/bin/afl-clang-fast"```
made it. FWIW:
```
ls -l /usr/bin/afl-*
-rwxr-xr-x 1 root root  42904 May 30 00:21 /usr/bin/afl-analyze
lrwxrwxrwx 1 root root     14 May 30 00:21 /usr/bin/afl-clang -> afl-clang-fast
lrwxrwxrwx 1 root root     14 May 30 00:21 /usr/bin/afl-clang++ -> afl-clang-fast
-rwxr-xr-x 1 root root  44224 May 30 00:21 /usr/bin/afl-clang-fast
lrwxrwxrwx 1 root root     14 May 30 00:21 /usr/bin/afl-clang-fast++ -> afl-clang-fast
-rwxr-xr-x 1 root root  14650 May 30 00:21 /usr/bin/afl-cmin
-rwxr-xr-x 1 root root  11993 May 30 00:21 /usr/bin/afl-cmin.bash
-rwxr-xr-x 1 root root 239480 May 30 00:21 /usr/bin/afl-fuzz
lrwxrwxrwx 1 root root      7 May 30 00:21 /usr/bin/afl-g++ -> afl-gcc
-rwxr-xr-x 1 root root  18216 May 30 00:21 /usr/bin/afl-gcc
-rwxr-xr-x 1 root root  14120 May 30 00:21 /usr/bin/afl-gotcpu
-rwxr-xr-x 1 root root   4989 May 30 00:21 /usr/bin/afl-plot
-rwxr-xr-x 1 root root  59264 May 30 00:21 /usr/bin/afl-showmap
-rwxr-xr-x 1 root root   3605 May 30 00:21 /usr/bin/afl-system-config
-rwxr-xr-x 1 root root  63360 May 30 00:21 /usr/bin/afl-tmin
-rwxr-xr-x 1 root root   7001 May 30 00:21 /usr/bin/afl-whatsup
```

**Comment by vanhauser-thc** [comment]
without pasting the failure messages etc - how should we know what went wrong?

**Comment by vanhauser-thc** [comment]
I remembered that we got a PR that changed the detection of the binary name. and there afl-clang missing. please test, the fix is in the dev branch.
in the future please write paste failure message so we can have a good chance to guess what the problem is.

**Comment by toralf** [comment]
https://github.com/AFLplusplus/AFLplusplus/pull/331/commits/82b6b8c87e1bc5394349fd2931b306f671c4da1a fixed it here

**Comment by toralf** [comment]
> I remembered that we got a PR that changed the detection of the binary name. and there afl-clang missing. please test, the fix is in the dev branch.
> in the future please write paste failure message so we can have a good chance to guess what the problem is.
I tried to cherry pick both commits but they didn't applied cleanly on top of 2.65c.
Will wait for a next release.

**Comment by vanhauser-thc** [comment]
just do a "git clone https://github.com/AFLplusplus/AFLplusplus  ; cd AFLplusplus ; git checkout dev ; make source-only install " and you are fine

**Comment by toralf** [comment]
I patched 2.65c with "git swithc dev; git diff 2.65c.. > /etc/portage/patches/app-forensics/AFLplusplus-2.65c/dev.patch"
and tested the excuteabled - seems to solve the issue.
